SigProfilerClusters/classifyFunctions.py

```diff
@@ -340,12 +340,34 @@ def pullVaf(project, project_path, variant_caller=None, correction=True):
     )
     vcf_files = [x for x in os.listdir(vcf_path) if x != ".DS_Store"]
 
+    # Dictionary for variant caller mapping
+    variant_type_dict = {
```

Comment:
Remove case sensitive keys because this syntax is used:
`if variant_caller.lower() in variant_type_dict`